avocado/utils/configure_network.py

```diff
@@ -200,3 +201,18 @@ def is_interface_link_up(interface):
         log.info("Interface %s link is up", interface)
         return True
     return False
+
+
+def get_ip_address(interface, version='4'):
+    """
+    Get the IP address from a network interface
+    :param interface: name of the interface
+    :param version: IP version, default '4'
+    :return: IP address or False
+    """
+    cmd = "ip -%s -j address show %s" % (version, interface)
+    output = json.loads(process.system_output(cmd).decode("utf-8"))
```

Comment:
I'm still not sure if you need the decode here. Are you having any problems with encoding?

Reply:
Sorry, yes it works fine without it :)